workflow/rocoto/rocoto.py

```diff
@@ -333,6 +335,31 @@ def _add_streq_tag(dep_dict: Dict[str, Any]) -> str:
     return string
 
 
+def _add_sh_tag(dep_dict: Dict[str, Any]) -> str:
+    """
+    create a simple shell execution tag
+    :param: dep_dict: shell command to execute
+    :type dep_dict: dict
+    :return: Rocoto simple shell execution dependency
+    :rtype: str
+    """
+
+    shell = dep_dict.get('shell', '/bin/sh')
```

Comment:
We should make the default `bash` to match the rest of workflow

Reply:
Rocoto's default shell is `sh`.  When rocoto submits jobs, it does so with
`/bin/sh`. IMO, the default should not be altered.